duva/src/domains/caches/cache_objects.rs

```diff
@@ -6,6 +6,20 @@ use bincode::{
 use chrono::{DateTime, Utc};
 use std::time::Duration;
 
+pub enum CacheValueType {
```

Comment:
Guess we wouldn't store data with tag like this? 

Say, if we want to support Vector type, what would be the meaning of this enum ?

Would that be something that we can learn after applying parsing logic? 

Then this `CacheValueType` will merely be the communication, not subject to domain. 

HDYT?